### DISCUSSION TITLE
Unlockable emoticons user page wallpapers and profile icons

### DISCUSSION THREAD

**Discussion by rabbids4eva** [RetroAchievements/RAWeb#772] [Ideas]
How it works: If you master a set or get a specific achievement on a supported game, you could earn a profile icon that you could change if you want, emoticons for forums and chat and/or wallpapers for your userpage, complete with unique details and even fonts.

**Comment by rabbids4eva** [comment]
Example: Atari 2600 Pixel Art Profile Icon = Earned by mastering at least 5 Atari 2600 sets
Pikachu Pixel Profile Wallpaper = Unlock "Just Like Ash" in Pokemon - Yellow Version

**Comment by eldexterr** [comment]
Kind of related to #769
